src/data/nat/choose/multinomial.lean

```diff
@@ -0,0 +1,171 @@
+/-
+Copyright (c) 2022 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+Heavily inspired by code from Kyle Miller and Kevin Buzzard
```

Comment:
We don't usually add such notice in the copyright header (correct me if I'm wrong), but you may add "Co-authored by : " lines in the PR description.

Reply:
@kmill Given your involvment I've added you as co-author. Feel free to let me know if you'd rather I remove that:)